src/StatsModelComparisons.jl

```diff
@@ -40,23 +41,25 @@ models. arXiv preprint arXiv:1507.04544.
 Aki Vehtari and Andrew Gelman (2015). Pareto smoothed importance sampling.
 arXiv preprint arXiv:1507.02646.
 """
-
 module StatsModelComparisons
 
-    using StatsFuns, Statistics
-
-    mc_path = @__DIR__
-
-    include("psisloo.jl")
-    include("psislw.jl")
-    include("gpdfitnew.jl")
-    include("gpinv.jl")
-    include("waic.jl")
-    include("pk_utilities.jl")
-    include("dic.jl")
-    include("legacy.jl")
-
-    export mc_path, deviance, dic, psisloo, gpdfitnew, gpinv
-    export waic, aic, bic
-
+using RecipesBase
+using StatsFuns
+using Statistics
+
+const mc_path = @__DIR__
+
+include("psisloo.jl")
+include("psislw.jl")
+include("gpdfitnew.jl")
+include("gpinv.jl")
+include("waic.jl")
+include("pk_utilities.jl")
+include("dic.jl")
+include("legacy.jl")
+
+export mc_path, deviance, dic, psisloo, gpdfitnew, gpinv
```

Comment:
It seems a bit strange to export `mc_path` - what is its purpose? In general, one can retrieve the directory with `dirname(pathof(StatsModelComparisons))` if StatsModelComparisons is loaded. In any case, probably the variable should be a constant.